modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js

```diff
@@ -45,6 +45,7 @@ export default function DropdownMenu({
 				menuElementAttrs={menuProps}
 				trigger={
 					<ClayButton
+						aria-label={Liferay.Language.get('actions')}
```

Comment:
This is going to affect too many instances of dropdowns and in places that might not be action menus or in situations where a label is provided and we don't need something extra for the screen reader. We should change this only for the location that the bug was reported. 

Additionally, it would be better to add this as a tooltip, rather than an `aria-label` so that it can benefit all users. Tooltips in Liferay are added using the `title` attribute and also adding the `lfr-portal-tooltip` class to the element.

Reply:
The taglib name is `dropdown-actions`, so I thought it would be ok to call it "actions" across the portal as a default. Ideally, attributes added on the taglib usage should overwrite the default one and we would be safe on the accessibility side, always having a generic title on the element, maybe "dropdown". But I am not sure about this behavior, so I went the way you described and added the attribute to the specific taglib usage in `/home/me/dev/projects/liferay-portal/modules/apps/journal/journal-web/src/main/resources/META-INF/resources/view_entries.jsp 
`

Unfortunately, adding a title attribute caused a bug where the tooltip shown is not the one on the button, but one on the entire entry element, and showing this tooltip changes the name of the property from "title" to "data-restore-title", making screen reader read "unlabeled" again. So, the quickest fix is adding a aria-label attribute and leave the element without a tooltip. Will send this change to the Echo team's repository.